README.md

```diff
@@ -55,18 +53,18 @@ Below are more specific and complete examples of the media query syntax:
 }
 ```
 
-When defining a media query, we use the `@media` keyword followed by conditional
-statements that will trigger the media query on or off. Below we will break down
-each property available in media queries:
+When defining a media query, we use the `@media` keyword followed by
+conditional statements that will trigger the media query to apply or not. Below
+we will break down each property available in media queries:
 
 #### `not`, `only`
 
 The logical keywords "not" or "only" can be used optionally to include or
 exclude specific media types or screen sizes. For instance, we may want to
 specify that a navigation bar should extend across the width of our page, but
-_only_ on screens larger than 800px in height. It might also be the case that
+_only_ on screens larger than `800px` in height. It might also be the case that
 you want a CSS rule to apply to all devices, but _not_ screens smaller than
-400px wide.
+`400px` wide.
 
 ```css
```

Comment:
The consistency is a little odd here. We're talking about two different media rules, and then giving an example of a third that's not only different, but has a very odd syntax in square brackets. I think I'd give an example of each media query, so the student can see the difference.

Reply:
Changed.